src/main/java/dev/jbang/cli/Export.java

```diff
@@ -619,6 +626,34 @@ String getJavaVersion(Project prj, boolean minorVersionFor8) {
 		}
 		return javaVersion >= 9 ? Integer.toString(javaVersion) : "1." + javaVersion;
 	}
+
+	void copyWrapperFile(String srcPath, String dstPath, boolean execFlag) throws IOException {
+		File dstFile = new File(dstPath);
+		File dstDir = new File(dstFile.getParent());
+
+		dstDir.mkdirs();
+		InputStream ifs = this.getClass().getResourceAsStream(srcPath);
+		Files.copy(ifs, Paths.get(dstPath), StandardCopyOption.REPLACE_EXISTING);
+
+		boolean isWindows = System.getProperty("os.name").toLowerCase().contains("windows");
+
+		if (isWindows)
+			return;
+
+		if (execFlag) {
```

Comment:
Ok, everything is perfect for merging, but .... I wonder if you could do me a favor and perform a small refactor: there is already some code in JBang to set a file as executable (See https://github.com/jbangdev/jbang/blob/main/src/main/java/dev/jbang/cli/App.java#L192). Could you perhaps extract that, move it to Util.java (just below the isExecutable() method for example) and make Export use that one as well?

Reply:
That's a good idea. I'll implemented the requested changes.